packages/voyager-server/package.json

```diff
@@ -22,6 +22,8 @@
   },
   "dependencies": {
     "@aerogear/voyager-audit": "^0.3.0",
+    "@aerogear/voyager-conflicts": "^0.3.0",
```

Comment:
Is this necessary inside actual deps or just dev deps?

Reply:
Idea is that all will be inside actual production dependencies as others.